src/cmdlib.sh

```diff
@@ -414,7 +414,8 @@ impl_rpmostree_compose() {
         # "cache2" has an explicit label so we can find it in qemu easily
         if [ ! -f "${workdir}"/cache/cache2.qcow2 ]; then
             qemu-img create -f qcow2 cache2.qcow2.tmp 10G
-            LIBGUESTFS_BACKEND=direct virt-format --filesystem=xfs --label=cosa-cache -a cache2.qcow2.tmp &&
+            source /usr/lib/coreos-assembler/libguestfish.sh
+            virt-format --filesystem=xfs --label=cosa-cache -a cache2.qcow2.tmp
```

Comment:
Let's do this in a subshell to be safe (i.e. wrapped in `(...)`). E.g. it uses an `EXIT` trap, which will override any previously set `EXIT` trap from any consumer of `cmdlib.sh`.